dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -24,6 +24,7 @@ void NotificationListenerManager::AddListener(std::wstring appId, std::wstring p
     {
         ComPtr<INotificationListener> listener;
         THROW_IF_FAILED(MakeAndInitialize<NotificationListener>(&listener, m_foregroundSinkManager, appId, processName));
+        THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(appId.c_str(), GUID_NULL));
```

Comment:
Is this addressing the case where RegisterForegroundActivator is called before we register a full trust app?

Reply:
Yes